queries/php/highlights.scm

```diff
@@ -2,39 +2,55 @@
 
 (variable_name) @variable
 
+; Constants
+
+((name) @constant
+ (#vim-match? @constant "^_?[A-Z][A-Z\d_]*$"))
+((name) @constant.builtin
+ (#vim-match? @constant.builtin "^__[A-Z][A-Z\d_]+__$"))
```

Comment:
looks like this could also be handled by the faster lua match?

Reply:
Thanks, I have added your suggestion.